### PR TITLE
FeatureForm composable KDoc edit

### PR DESCRIPTION
**Pull request by sorenoid**
Just a bit of cleanup based on suggestions from @gunt0001

### CODE REVIEW

**Review by kaushikrw — APPROVED** [review]

File: toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/FeatureForm.kt [hunk 1]
```diff
@@ -93,8 +93,8 @@ public sealed class ValidationErrorVisibility {
 
 /**
  * A composable Form toolkit component that enables users to edit field values of features in a
- * layer using forms that have been configured externally (using either in the the Web Map Viewer
- * or the Fields Maps web app).
+ * layer using forms that have been configured externally. Forms may be configured in the Web Map Viewer
+ * or Fields Maps Designer).
```
Comment (gunt0001) [inline toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/FeatureForm.kt:97]: Would linking make sense here? I'm ok if you prefer not to for some reason...
```suggestion
 * layer using forms that have been configured externally. Forms may be configured in the [Web Map Viewer](https://www.arcgis.com/home/webmap/viewer.html)
 * or [Fields Maps Designer](https://www.arcgis.com/apps/fieldmaps/)).
```

Comment (gunt0001) [reply toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/FeatureForm.kt:97]: Can forms be configured in ArcGIS Pro? If so, I would add that as well.

**Review by gunt0001 — APPROVED** [review]
@sorenoid - some suggestion/question.

File: toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/FeatureForm.kt (hunk 1 above)
Comment (sorenoid) [reply toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/FeatureForm.kt:97]: nope

**Pull request merged**